src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js

```diff
@@ -0,0 +1,59 @@
+import React, { useEffect, useState } from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import Loading from '../../Loading/Loading';
+
+const ReportsTrafficSources = function({
+  domainName,
+  dateFrom,
```

Comment:
We should use also `dateTo`, does DataHub not filter by end date?

Reply:
DataHub only need `dateFrom` i talk with Leo

Reply:
Only to clarify... DataHub **allow** send only `dateFrom`, `dateTo` will be **now**; but you can send `dateTo` too.